quickstart/backend/src/main/java/com/digitalasset/quickstart/service/UserApiImpl.java

```diff
@@ -55,9 +55,9 @@ public CompletableFuture<ResponseEntity<AuthenticatedUser>> getAuthenticatedUser
         // Create the AuthenticatedUser object
         AuthenticatedUser user = new AuthenticatedUser(
                 // name
-                party.split("::")[0],
+                auth.getPrincipal().getAttribute("name"),
                 // party
-                party,
+                auth.getPrincipal().getAttribute("party"),
```

Comment:
How does this exactly work? How is that "party" attribute set and where does it come from?

Reply:
I now see it: it is put in as an extra claim by the `...AuthSuccessHandler` above: https://github.com/digital-asset/cn-quickstart/pull/25/files#diff-53db2ce42e2ac7234eabf3cb164afdecb988c65ced2a18989384879509093eddR57

Consider making this an `_virtual_claim_client_party_id` so it is easily grepable.